backend/test_auth.py

```
@@ -0,0 +1,55 @@
+        print(f"Signup Error: {e}")
+        return False
+
+def test_login():
+    """Test user login."""
+    payload = {
+        "email": "test@example.com",
+        "password": "testpassword123"
+    }
+
+    try:
+        response = requests.post(f"{BASE_URL}/auth/login", json=payload)
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)